app/js/components/common/notification/notification.component.js

```diff
@@ -0,0 +1,43 @@
+const template = require("./notification.tpl.html");
+
+const NotificationComponent = {
+  template,
+  controller: [
+    "$log",
+    "$rootScope",
+    "$scope",
+    "$timeout",
+    function($log, $rootScope, $scope, $timeout) {
+      "ngInject";
+      $log = $log.getInstance("NotificationComponent", true);
+      $log.log("");
+
+      let vm = this;
+
+      vm.removeNotification = () => {
+        $log.log(`removed`);
+        vm.message = "";
+        vm.type = "";
+      };
+
+      $rootScope.$on("TRIGGERED_NOTIFICATION", (evt, data) => {
+        const { message, type } = data;
+        $log.log(`triggered: ${type} message=${message}`);
+
+        vm.message = message;
+        vm.type = type;
+
+        $scope.$apply();
+
+        $timeout(() => {
+          vm.removeNotification();
+        }, 5000);
```

Comment:
I'm not entirely sure why we want notifications to dismiss themselves, but that's more of a UX question that maybe the Cognoma team already settled. Just flagging here to be *sure* that's what we want.

Reply:
Hmm, lemme ask Deepa about that.

Reply:
She said it can dismiss itself on it's own 👍